.clang-format

```diff
@@ -92,3 +92,4 @@ UseTab: ForContinuationAndIndentation
 WhitespaceSensitiveMacros:
   - STRINGIFY
   - Z_STRINGIFY
+AlignEscapedNewlines: Right
```

Comment:
This needs to be in a separate commit

Reply:
My bad, this was just a test, not supposed to be in this PR haha